packages/fluentui/react-component-nesting-registry/src/index.ts

```diff
@@ -1,2 +1,2 @@
-export { default as Unstable_NestingAuto } from './NestingAuto';
```

Comment:
default exports in `react-component-nesting-registry`:
![image](https://user-images.githubusercontent.com/9615899/86406440-65b8d000-bcb3-11ea-86b5-435a2694ffbd.png)

Reply:
Fixed 👍